packages/desktop-client/src/components/common.tsx

```diff
@@ -8,19 +8,11 @@ import { colors } from '../style';
 import Button from './common/Button';
 
 export { default as AnchorLink } from './common/AnchorLink';
-export { default as Block } from './common/Block';
 export { default as Button, ButtonWithLoading } from './common/Button';
-export { default as Card } from './common/Card';
 export { default as Select } from './common/Select';
-export { default as InitialFocus } from './common/InitialFocus';
 export { default as Input } from './common/Input';
-export { default as Label } from './common/Label';
-export { default as Menu } from './common/Menu';
-export { default as Modal, ModalButtons } from './common/Modal';
-export { default as Stack } from './common/Stack';
 export { default as Text } from './common/Text';
 export { default as View } from './common/View';
```

Comment:
Do you want to remove these too?

Reply:
Yea. In pt3 those will go too. Didn't want to do one massive migration, so doing this slowly over multiple PRs.

Reply:
Makes sense, it was a bit surprising that there were only 40 files using Text/View. Guess that’s because there are way more!